odufrn_downloader/mixins/filters/LevenshteinMixin.py

```diff
@@ -45,17 +46,24 @@ def search_related(self, keyword: str, input_list: list) -> list:
             palavra-chave com a qual será feita a busca.
         input_list: list
             lista com os valores que irá verificar a similaridade com keyword.
+        split: bool
+            flag que indica se a palavra-chave deve ser dividida
 
         Retorno
         -------
         lista de valores com nome similares à palavra de interesse.
         """
         filter_list = []
         for item in input_list:
-            for word in item.split('-'):
-                ratio = self.levenshtein(
-                    [k for k in keyword], [d for d in word]
-                )
+            str1 = [k for k in keyword]
```

Comment:
Se a `keyword` é constante, porque ficar sempre separando ela em cada iteração? Não seria melhor declarar esse array fora do `for`?